with-animated-splash-screen/App.js

```diff
@@ -0,0 +1,130 @@
+import { AppLoading, SplashScreen, Updates } from 'expo';
+import { Asset } from 'expo-asset';
+import Constants from 'expo-constants';
+import React from 'react';
+import { Animated, Button, StyleSheet, Text, View } from 'react-native';
+
+// Instruct SplashScreen not to hide yet, we want to do this manually
+SplashScreen.preventAutoHide();
+
+export default function App() {
+  return (
+    <AnimatedAppLoader image={{ uri: Constants.manifest.splash.image }}>
```

Comment:
Used `Constants` here, I see you also used some other variables from that. Not sure if we should just hardcode the URL for the sake of simplicity?

Reply:
either way is fine